backend/wmg/api/v1.py

```diff
@@ -353,10 +353,17 @@ def _add_missing_combinations_to_gene_expression_df_for_rollup(
         missing_combinations = available_combinations.difference(available_combinations_per_gene)
         for combo in missing_combinations:
             entry = {dim: combo[i] for i, dim in enumerate(group_by_terms)}
-            entry.update({col: 0 for col in numeric_columns})
-            entry["n_cells_tissue"] = n_cells_tissue_dict[entry["tissue_ontology_term_id"]]
```

Comment:
Thanks for catching this, Prathap!! I think the simplest fix would be to replace this line with:

```python
entry["n_cells_tissue"] = n_cells_tissue_dict.get(entry["tissue_ontology_term_id"],0)
```

What do you think?

Reply:
@atarashansky - Even better! I will make this fix

Reply:
@atarashansky - On second thought, I see a couple of disadvantages here. We get the advantage of conciseness with your suggestion but we also get a couple of disadvantages (as I see it):

1. _Loss of clarity_ - It is not clear that the combos involving the tissue will be removed. In fact, a bug in a function further up the callstack that manipulates the dataframe returned from this function could mess things up. The current approach limits that possibility by not including these zero entries in dataframe.
2. _Greater memory footprint that might negatively affect performance_ - the memory footprint of the dataframe that is returned is higher by including the combos that will be eliminated anyway. The extra memory is 

```
|genes with expression values| x |tissues_with_no_gene_expression| x |compare_dim|
```

What do you think?

Reply:
I'll leave it up to you. More clarity is indeed better.  Also, for what it's worth, I think the extra memory used by the 0's that would be added in this edge case is negligible.